### PR TITLE
Add Wox

### PR DESCRIPTION
**Pull request by Reset816**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/565)

### CODE REVIEW

File: manifests/Wox/Wox/1.4.1129.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Id: Wox.Wox
+Version: 1.4.1129
```
Comment (ousugo) [inline manifests/Wox/Wox/1.4.1129.yaml:2]:
```suggestion
Version: 1.4.1196
```

File: manifests/Wox/Wox/1.4.1129.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Author: Wox-launcher
+License: MIT
+LicenseUrl: https://github.com/Wox-launcher/Wox/blob/master/LICENSE
+AppMoniker: wox
+Tags: launcher
+Description: Launcher for Windows, an alternative to Alfred and Launchy.
+Homepage: https://github.com/Wox-launcher/Wox
+Installers:
+  - Arch: x86
+    Url: https://github.com/Wox-launcher/Wox/releases/download/v1.4.1129/Wox-1.4.1129.exe
```
Comment (ousugo) [inline manifests/Wox/Wox/1.4.1129.yaml:14]:
```suggestion
    Url: https://github.com/Wox-launcher/Wox/releases/download/v1.4.1196/Wox-1.4.1196.exe
```

File: manifests/Wox/Wox/1.4.1129.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    Sha256: 57CA016A2900CDDFB371D9761C1C30DB6CC0DEF2E6CA81B1CDF7AA2CEF63748F
```
Comment (ousugo) [inline manifests/Wox/Wox/1.4.1129.yaml:15]:
```suggestion
    Sha256: 2cd868e2f83b629952be6ff4c207608ae7da4a74fabe512b138fa72261f526c4
```

File: manifests/Wox/Wox/1.4.1129.yaml (hunk 1 above)
Comment (ousugo) [inline manifests/Wox/Wox/1.4.1129.yaml:2]:
```suggestion
Version: 1.4.1196
```

File: manifests/Wox/Wox/1.4.1129.yaml (hunk 2 above)
Comment (ousugo) [inline manifests/Wox/Wox/1.4.1129.yaml:14]:
```suggestion
    Url: https://github.com/Wox-launcher/Wox/releases/download/v1.4.1196/Wox-1.4.1196.exe
```

File: manifests/Wox/Wox/1.4.1129.yaml (hunk 3 above)
Comment (ousugo) [inline manifests/Wox/Wox/1.4.1129.yaml:15]:
```suggestion
    Sha256: 2cd868e2f83b629952be6ff4c207608ae7da4a74fabe512b138fa72261f526c4
```

**Review by ousugo — CHANGES_REQUESTED** [review]
Change to add latest version V1.4.1196, and I submitted it to Windows Defender.

**Comment by Reset12138** [comment]
> Change to add latest version V1.4.1196, and I submitted it to Windows Defender.

thank you

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**